test/jdk/java/awt/dnd/ImageDecoratedDnD/ImageDecoratedDnD.java

```diff
@@ -21,177 +21,85 @@
+                over the drop targets with activated extended OLE DnD support
+                (that are, the desktop and IE).
+
+            3. Release the mouse button.
+
+                The panel should turn red again and a yellow button labeled,
+                "Drag ME!" should appear inside the panel. You should be able,
+                to repeat this operation multiple times.
+
+            If above is true press PASS, else press FAIL.
+            """;
+
+    public static void main(String[] args) throws Exception {
+        PassFailJFrame.builder()
+                .title("Test Instructions")
+                .instructions(INSTRUCTIONS)
+                .rows((int) INSTRUCTIONS.lines().count() + 2)
+                .columns(38)
+                .testUI(ImageDecoratedDnD::createUI)
+                .build()
+                .awaitAndCheck();
+    }
 
-public class ImageDecoratedDnD extends Applet {
-    //Declare things used in the test, like buttons and labels here
-
-    public void init() {
-        //Create instructions for the user here, as well as set up
-        // the environment -- set the layout manager, add buttons,
-        // etc.
-        this.setLayout(new BorderLayout());
-
-        String[] instructions =
-                {
-                        "A Frame, which contains a yellow button labeled \"Drag ME!\" and ",
-                        "a red panel, will appear below. ",
-                        "1. Click on the button and drag to the red panel. ",
-                        "2. When the mouse enters the red panel during the drag, the panel ",
-                        "should turn yellow. On the systems that supports pictured drag, ",
-                        "the image under the drag-cursor should appear (ancor is shifted ",
-                        "from top-left corner of the picture inside the picture to 10pt in both dimensions ). ",
-                        "In WIN32 systems the image under cursor would be visible ONLY over ",
-                        "the drop targets with activated extended OLE D\'n\'D support (that are ",
-                        "the desktop and IE ).",
-                        "3. Release the mouse button.",
-                        "The panel should turn red again and a yellow button labeled ",
-                        "\"Drag ME!\" should appear inside the panel. You should be able ",
-                        "to repeat this operation multiple times."
-                };
-        Sysout.createDialogWithInstructions(instructions);
-
-    }//End  init()
-
-    public void start() {
-        Frame f = new Frame("Use keyboard for DnD change");
+    public static Frame createUI() {
+        Frame frame = new Frame("Ctrl + Drag - Image DnD test");
         Panel mainPanel;
         Component dragSource, dropTarget;
 
-        f.setBounds(0, 400, 200, 200);
-        f.setLayout(new BorderLayout());
+        frame.setBounds(0, 400, 400, 400);
```

Comment:
```suggestion
        frame.setSize(400, 400);
```
The frame is positioned by `PassFailJFrame`, so setting the location is redundant.